src/router.dispatch.ts

```diff
@@ -0,0 +1,29 @@
+// Do Router dispatch here, i.e. map incoming routes to appropriate router
```

Comment:
This file has been deleted in the new structure. Hence should be deleted please.